src/main/java/io/appform/dropwizard/sharding/healthcheck/BlacklistingAwareHealthCheck.java

```diff
@@ -24,9 +31,14 @@ protected Result check() {
             log.info("returning healthy since shard is blacklisted [{}]", shardId);
             return Result.healthy();
         }
+
```

Comment:
This should be the first thing to be checked

Reply:
Done